src/app/users/users.component.html

```diff
@@ -1,23 +1,26 @@
-<button pButton type="button" label="Add new user" class="p-button-success" (click)="openForm('Add')" ></button>
+<button pButton type="button" label="Add new user" class="p-button-success" (click)="openForm('Add')"></button>
 
 <button *ngIf="selectedUsers.length" pButton pRipple type="button" label="Delete selected" class="p-button-danger"
-	(click)="openDeleteSelectedUsersDialog()" ></button>
+	(click)="openDeleteSelectedUsersDialog()"></button>
 
-<p-table styleClass="p-datatable-striped"[columns]="columns" [value]="users" selectionMode="checkbox" [(selection)]="selectedUsers" [loading]="isLoading" >
-	<ng-template pTemplate="header">
+<p-table styleClass="p-datatable-striped" [columns]="columns" [value]="users"
+	selectionMode="checkbox" [(selection)]="selectedUsers" [loading]="isLoading" [paginator]="true"
+	[rows]="5" [rowsPerPageOptions]="[5,10,15]" >
+	<ng-template pTemplate="header" let-columns>
 		<tr>
 			<th style="width: 2%">
-				<p-tableHeaderCheckbox ></p-tableHeaderCheckbox>
+				<p-tableHeaderCheckbox></p-tableHeaderCheckbox>
 			</th>
-			<th *ngFor="let column of columns" [ngStyle]="{'width': column.width}">
-				{{column.header}}
+			<th sortable="true" *ngFor="let column of columns"
+				[ngStyle]="{'width': column.width}" [pSortableColumn]="column.field" >
+				{{column.header}} <p-sortIcon  *ngIf="column.field!='actions'" field="column.field"></p-sortIcon>
```

Comment:
did you try if the sorting is available on the actions column? even if the icon isn't here
put p-icon on new line

Reply:
hint pSortableColumnDisabled